main.cpp

```diff
@@ -243,238 +242,229 @@ struct Resturant
     std::string brand = "Fender";
-    //2) model number (int)
     int modelNum = 1200;
-    //3) power (float)
     float power = 200.f;
-    //4) impedence (float)
     float impedence = 200.f;
-    //5) outputs (int)
     int outputs = 4;
-    //1) project audio signal
     void amplify();
-    //2) power speakers
     void speakers();
-    //3) protect system from failure
     void surgeProtect();
 };
-/*
-Thing 7) receiver
-5 properties:
-    1) brand (std::string)
-    2) model number (int)
-    3) functions (int)
-    4) inputs (int)
-    5) outputs (int)
-3 things it can do:
-    1) play cds
-    2) play records
-    3) play radio
- */
+
+void Amp::amplify()
+{
+    
+}
+
+void Amp::speakers()
+{
+    
+}
+
+void Amp::surgeProtect()
+{
+    
+}
+
 struct Receiver
 {
-    //1) brand (std::string)
     std::string brand = "Pioneer";
-    //2) model number (int)
     int modelNum = 2600;
-    //3) functions (int)
     int function = 6;
-    //4) inputs (int)
     int inputs = 5;
-    //5) outputs (int)
     int outputs = 4;
-    //1) play cds
     void playCds();
-    //2) play records
     void playRecords();
-    //3) play radio
     void playRadio();
 };
-/*
-Thing 8) remote
-5 properties:
-    1) brand (std::string)
-    2) model number (int)
-    3) range (float)
-    4) function selector (int)
-    5) batteries needed (int)
-3 things it can do:
-    1) select inputs
-    2) adjust volume
-    3) select mode
- */
+
+void Receiver::playCds()
+{
+    
+}
+
+void Receiver::playRecords()
+{
+    
+}
+
+void Receiver::playRadio()
+{
+    
+}
+
 struct Remote
 {
-    //1) brand (std::string)
     std::string brand = "Pioneer";
-    //2) model number (int)
     int modelNum = 1600;
-    //3) range (float)
     float range = 135.4f;
-    //4) function selector (int)
     int selector = 5;
-    //5) batteries needed (int)
     int batteries = 2;
-    //1) select inputs
     void inputs();
-    //2) adjust volume
     int adjustVol(int volUp, int volDwn, int currVol);
-    //3) select mode
     void mode();
 };
-/*
-Thing 9) turntable
-5 properties:
-    1) brand (std::string)
-    2) model number (int)
-    3) belt speed (float)
-    4) outputs (int)
-    5) needle weight (float)
-3 things it can do:
-    1) plays records
-    2) plays at 33 or 45 rpms
-    3) adjusts for anti-skip
- */
+
+void Remote::inputs()
+{
+    
+}
+
+int Remote::adjustVol(int volUp, int volDwn, int currVol)
+{
+    return currVol += volUp - volDwn;
+}
+
+void Remote::mode()
+{
+    
+}
+
 struct Turntable
 {
-    //1) brand (std::string)
     std::string brand = "Mercury";
-    //2) model number (int)
     int modelNum = 1400;
-    //3) belt speed (float)
     float beltSpeed = 25.4f;
-    //4) outputs (int)
     int outputs = 4;
-    //5) needle weight (float)
     float needleWeight = 0.0043f;
-    //1) plays records
     void playRecord();
-    //2) plays at 33 or 45 rpms
     bool rpms(bool thirtThree);
-    //3) adjusts for anti-skip
     void skipAdjust();
```

Comment:
I didn't catch these improper function names in part 1. 
What you have written here does not match your plain-english actions.
You need to include the action words in your function names
also, your 2nd action is poorly named.  It should be 'choose playback speed'
```
bool choosePlaybackSpeed(bool use33Rpm);
void adjustForAntiSkip();
```
- [ ] revise